### ISSUE TITLE
Not working for jupyterlab, I think

### ISSUE THREAD

**Issue by SamuelGabriel** [alexi/juno#12]
Hi there,

I tried to install Juno for jupyterlab, but got 

```
Javascript Error: Jupyter is not defined
```

even after updating jupyterlab, ipython, ipympl, ipywidgets, nodejs-bin.

Could it be that this is due to Juno only working for Jupiter notebooks, not lab?

**Comment by amauboussin** [comment]
That's correct, unfortunately it only works for normal Jupyter notebooks right now :(
